Software/Development/SemesterProject/App/queueObject.c

```diff
@@ -129,10 +130,14 @@ void UARTIntPrint(char *variableName, int value)
 {
     char str[10];
     char *strASCII;
-    strASCII = intToAscii(value, str);
-    UARTStringPrint(variableName);
-    UARTStringPrint(strASCII);
-    UARTStringPrint("\n\r");
+    if (!isUARTBufferFull())
```

Comment:
this is wrong in the sense, that the logic questions whther there's a single space in the queue, but then adds N variables to the queue: NOT GOOd

Reply:
Then fix it 😉